deploy/charts/rook-ceph/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if and .Values.serviceMonitor.enabled (or .Values.csi.enableGrpcMetrics .Values.csi.enableLiveness) }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: csi-metrics
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "library.rook-ceph.labels" . | nindent 4 }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
+  selector:
+    matchLabels:
+      app: csi-metrics
+  endpoints:
+    - port: csi-http-metrics
+      path: /metrics
+      interval: {{ .Values.serviceMonitor.interval }}
+    - port: csi-grpc-metrics
```

Comment:
GRPC metrics are disabled by default. If we're going to enable this endpoint, we need to check for the helm setting: `csi.enableGrpcMetrics`